packages/compartment-mapper/test-manual/policy/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "endo-sample",
```

We try to use `@endo/...` scoping so that those who are watching don't get nervous about NPM hijacking.

```suggestion
  "name": "@endo/compartment-mapper-demo-policy",
```

---

And/or a `scripts.preinstall` that dies loudly.